Makefile

```diff
@@ -638,6 +642,7 @@ $(PY_PROTO_INIT): | $(PY_PROTO_BUILD_DIR)
 	touch $(PY_PROTO_INIT)
 
 clean:
+	@- $(RM) -rf $(CUB_DIR)
```

Comment:
Perhaps this should go in `superclean`, or just be omitted? It's not that uncommon to `make clean` and seems a bit odd to me to force a re-download upon doing so.